lib/ramble/ramble/experimental/uploader.py

```diff
@@ -161,6 +161,27 @@ def format_data(data_in):
     return results
 
 
+def results_file_import(filename):
+    """
+    Import Ramble experiment results from a JSON file.
+    """
+    tty.debug("Import file")
```

Comment:
We print "import file" twice, can we reduce that somehow?

Reply:
Adjusted this to clarify that it's listing the filename for debuggin purpose (mirroring how it's done on lines 239-240)